### ISSUE TITLE
-autoload is not sufficient to attach any combination of cartridge/disk/prg file

### ISSUE THREAD

**Issue by mrdudz** [davidhorrocks/hoxs64#7]
Often you want to eg mount a cartridge (say, action replay) and then autorun a prg file. using autoload twice on the cmdline will not do the trick :)

(That said, it will perhaps still not work with actual action replay - because of how it has that menu that you need to skip with one of the f-keys. But generally, mounting a cartridge first, then run a .prg file should work - at least it would be needed for the test bench to work fully :))

**Comment by mrdudz** [comment]
To make that more clear, what the testbench is doing is basically this:
```
hoxs64.exe -defaultsettings -autoload ../C64/carts/aracidtest//ardummy.crt -autoload ../C64/carts/aracidtest//test-ramdetect-ar.prg
```
I expected that this would mount the cartridge (which is a ROM containing test patterns in this case) and then run the prg. However, what apparently happens is that it always just "autorun"s whatever it sees first on the cmdline, and the second one is ignored. Perhaps it would be an idea to have a seperate "mount" option for cartridge images :)

**Comment by davidhorrocks** [comment]
The next version will include a **-mountcart** command line option to work like this:

example:
`hoxs64.exe -autoload ..\C64\carts\aracidtest\test-ramdetect-ar.prg -mountcart ..\C64\carts\aracidtest\ardummy.crt`

-autoload will continue to support one file only.  autoload of a .PRG can be used with other options such as -mountdisk
I may in future include a way to autoload a stand-a-lone .PRG file via a temporary 1541 disk, as alternative to the current hacked .PRG file memory injection. This is already possible if the PRG is hosted on a .D64 file.

Only one cartridge is supported. I may add REU + another cart in future.

**Comment by mrdudz** [comment]
Nice! That sounds good :) I hope i will remember it next year :)

**Comment by davidhorrocks** [comment]
Fixed in v1.1.2.4

A command like this should mount a cartridge and load a prg file. 

`hoxs64.exe -autoload "..\C64\carts\aracidtest\test-ramdetect-ar.prg" -mountcart "..\C64\Game_Images\testprogs\C64\carts\aracidtest\ardummy.crt"`

**Comment by mrdudz** [comment]
cool, testbench updated too :)
